CodingPractice/PythonAssignments/cleancode/EmailExtract.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+
+class MalformedEmailAddressException(Exception):
+    pass
+
+
+class EmailDetails:
+    def __init__(self, _id, _domain):
+        self.id = _id
```

Comment:
Ooh dear.
You have obviously not understood the concept of using _ for instance variables.

[Read all about it here](https://stackoverflow.com/questions/6930144/underscore-vs-double-underscore-with-variables-and-methods)